core/server/middleware/index.js

```diff
@@ -50,9 +54,31 @@ middleware = {
     }
 };
 
-setupMiddleware = function setupMiddleware(blogApp, adminApp) {
+setupMiddleware = function setupMiddleware(blogApp) {
     var logging = config.logging,
-        corePath = config.paths.corePath;
+        corePath = config.paths.corePath,
+        adminApp = express(),
+        adminHbs = hbs.create();
+
+    // ##Configuration
+
+    // enabled gzip compression by default
+    if (config.server.compress !== false) {
+        blogApp.use(compress());
+    }
+
+    // ## View engine
+    // set the view engine
+    blogApp.set('view engine', 'hbs');
+
+    // Create a hbs instance for admin and init view engine
+    adminApp.set('view engine', 'hbs');
+    adminApp.engine('hbs', adminHbs.express3({}));
+
+    // Load helpers
+    helpers.loadCoreHelpers(adminHbs);
+
+    // end of moved code
```

Comment:
Nah, that was an artefact of what I intended to do next & was too much for one small coding session 😉 

Removed it now.